### PR TITLE
Issue 8306, updated masters recommendations

### PR DESCRIPTION
**Pull request by ahardin-rh**
Addresses https://github.com/openshift/openshift-docs/issues/8306

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@jeremyeder PTAL. Thanks!

**Comment by ahardin-rh** [comment]
@mffiedler @smarterclayton FYI

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -140,6 +142,9 @@ a master host in an {product-title} cluster of 2000 pods would be the minimum
 requirements of 2 CPU cores and 16 GB of RAM, plus 2 CPU cores and 3 GB of RAM,
 totaling 4 CPU cores and 19 GB of RAM.
 
+If etcd is co-located, there is just no way 2 cores will work, and probably 4 will
+be too little for even a moderately sized/loaded cluster.
+
```
Comment (mffiedler) [inline install_config/install/prerequisites.adoc]: Just a nitpick, but this statement and the similar bullet in Minimum Requirements seem redundant.    Suggest removing or maybe rewording it to "Masters with a co-located etcd require a minimum of 4 cores.  2 core systems will not work."

**Comment by ahardin-rh** [comment]
@mffiedler Updated. Thanks!
